src/Microsoft.ML.Data/Transforms/ColumnSelecting.cs

```diff
@@ -140,12 +138,12 @@ public sealed class ColumnSelectingTransformer : ITransformer
 
         bool ITransformer.IsRowToRowMapper => true;
 
-        public IEnumerable<string> SelectColumns => _selectedColumns.AsReadOnly();
+        internal IEnumerable<string> SelectColumns => _selectedColumns.AsReadOnly();
 
-        public bool KeepColumns { get; }
+        internal bool KeepColumns { get; }
 
-        public bool KeepHidden { get; }
-        public bool IgnoreMissing { get; }
+        internal bool KeepHidden { get; }
+        internal bool IgnoreMissing { get; }
```

Comment:
Do we really need to hide them? They are some direct states of public options.

Reply:
Maybe it just me been paranoid and I want to hide everything I can.
On other hand we need to provide this information for each transformer, like term dictionary or list of columns we been concatenating for concat, and we don't have conclusion how to do it yet.